src/pages/Product/components/ColorButton/ColorButton.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import "./ColorButton.scss";
+
+const ColorButton = (props) => {
+  return (
+    <button
+      className={`colorButton ${props.color}`}
+      onClick={() => {
+        props.setColor(props.color);
+      }}
+    />
+  );
+};
```

Comment:
```suggestion
const ColorButton = ({color, setColor}) => {
  return (
    <button
      className={`colorButton ${color}`}
      onClick={() => {
        setColor(color);
      }}
    />
  );
};
```
wrap up 시간에 배웠었던 객체 구조분해 할당을 이렇게 적용할 수 있겠습니다!